libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/NavigationView.java

```diff
@@ -590,7 +604,7 @@ private void updateInstructionMutedState(boolean isMuted) {
     int paddingBuffer = (int) resources.getDimension(R.dimen.route_overview_buffer_padding);
     int instructionHeight = (int) (resources.getDimension(R.dimen.instruction_layout_height) + paddingBuffer);
     int summaryHeight = (int) resources.getDimension(R.dimen.summary_bottomsheet_height);
-    return new int[] {leftRightPadding, instructionHeight, leftRightPadding, summaryHeight};
+    return new int[]{leftRightPadding, instructionHeight, leftRightPadding, summaryHeight};
```

Comment:
NIT 👀 how to set up Mapbox checkstyle here https://github.com/mapbox/mapbox-gl-native/wiki/Setting-up-Mapbox-checkstyle

Reply:
Done